### DISCUSSION TITLE
Private Tangle

### DISCUSSION THREAD

**Discussion by fishotw** [gohornet/hornet#687] [Q&A] [answered]
Hello, I already set up a coordinator private hornet node and two extra hornet node in individual Linux servers. I'm wondering that how to connect them all? Thanks a lot.

**Comment by luca-moser** [comment, marked as answer]
You have to adjust the `peering.json` of the nodes in order to let them connect to each other.

**Reply by fishotw** [reply]
Thank you very much. It works well.
